src/mplfinance/plotting.py

```diff
@@ -267,7 +283,10 @@ def plot( data, **kwargs ):
             mavc = None
             
         for mav in mavgs:
-            mavprices = data['Close'].rolling(mav).mean().values 
+            if ptype == 'renko':
+                mavprices = brick_values.rolling(mav).mean().values
```

Comment:
What I'm about to say here is more a matter of style and preference, than any kind of a good code versus better code.  So I will just lay out my thoughts and preferences here, and let you decide.

Among the extra return values from `_construct_renko_collections()`, two (new_dates, volumes) are lists, and one (brick_values) is now a dataframe.  In my mind, that's one vote towards leaving brick_values as a list, for a little more consistency.  Furthermore the *only* reason pandas is now included in `_utils.py` is to be able to return brick_values as a dataframe.  If it was needed in `_utils.py` for some other significant pandas work, that would be one thing, but since it is not, for me, that's another vote to return brick_values as a list (and remove pandas from `_utils.py`).

All this means that here, the line of code becomes

    mavprices = pd.Series(brick_values).rolling(mav).mean().values

which, on the one hand, is inconsistent with the fact that we already have (and don't need to construct) a Series on the `else:` side of this `if`.  

On the other hand, constructing the Series here *shows explicitly, by the code itself,* why `brick_values` is being converted to a Series; whereas inside `_construct_renko_collections()` its not clear at all why `brick_values` is being converted to a DataFrame but `new_dates` and `volumes` are not.

Balancing the benefit of the easily understood conversion to a Series here, plus the consistency of returning only lists from `_construct_renko_collections()`, against the inconsistency here compared with its `else:`, my preference is to tolerate the slight inconsistency here, in favor of returning brick_values as a list from `_construct_renko_collections()` and removing pandas from `_utils.py`.

Reply:
Yes, I agree, this makes the most sense.  Moving the conversion with pandas into plot() makes the code more understandable and avoids pointless imports!  Thank you for the suggestion!